app/services/foreman/renderer/scope/macros/base.rb

```diff
@@ -124,8 +124,12 @@ def save_to_file(filename, content, verbatim: false)
               content = Base64.encode64(content)
               "cat << #{delimiter} | base64 -d > #{filename}\n#{content}#{delimiter}"
```

Comment:
This case will still have an issue with `indent()`.

Reply:
Do you think we will have `\n` symbols there? I can implement the same solution here too

Reply:
Changed the verbatim option to stop using heredoc too